README.md

```diff
@@ -27,6 +27,38 @@ npm install eslint eslint-plugin-package-json jsonc-eslint-parser --save-dev
 
 ## Usage
 
+### Flat Config
+
+In your ESLint configuration file:
+
+```js
+import packageJson from "eslint-plugin-package-json/configs/recommended";
+
+export default [
+	// your other ESLint configurations
+	packageJson,
+];
+```
+
+If you want to override the recommended rules:
+
+```js
+import packageJson from "eslint-plugin-package-json/configs/recommended";
+
+export default [
+	// your other ESLint configurations
+	{
+		...packageJson,
+		rules: {
+			...packageJson.rules,
+			"package-json/valid-package-def": "off",
+		},
+	},
+];
+```
+
```

Comment:
[Docs] Since the legacy config has a mention of ESLint docs, I feel like it'd be nice to mention one here too. Maybe:

```suggestion

See [ESLint's _Configuration Files_ guide](https://eslint.org/docs/latest/use/configure/configuration-files-new) for details on how to customize your rules and other config settings.

```